### PR TITLE
address some readability/DRY concerns in train.py

### PR DESCRIPTION
**Pull request by skondrashov**
Hi all, I just started reading this project and a lot of the code seems harder to understand than I think it needs to be. I've applied some of my favorite code style ideas, things that seem to me to improve the readability and maintainability of the code:
```
1) Don't repeat yourself
2) Don't initialize names that are only used once
3) Expand complex nested statements onto multiple lines
```
 If this is helpful, I'll submit requests for other files I clean up in my fork!

### CODE REVIEW

File: tf/train.py [hunk 1]
```diff
@@ -281,6 +265,5 @@ def read(x):
                            type=str,
                            help='file to store weights in')
 
-    #mp.set_start_method('spawn')
```
Comment (tkondrashov) [inline tf/train.py]: not sure if this was supposed to be commented?

File: tf/train.py [hunk 2]
```diff
@@ -69,44 +56,51 @@ def get_latest_chunks(path, num_chunks, allow_less):
...
+    rule50_plane = tf.expand_dims(rule50_plane, -1)
+    rule50_plane = tf.tile       (rule50_plane, [1, 1, 8, 8])
+    rule50_plane = tf.cast       (rule50_plane, tf.float32)
+    rule50_plane = tf.divide     (rule50_plane, 99.)
+
+    # Zero plane and one plane.
     zero_plane = tf.zeros_like(rule50_plane)
     one_plane = tf.ones_like(rule50_plane)
+
     # For FRC unit planes must be replaced with 0 and 2 merged, 1 and 3 merged, two zero planes and then 4.
```
Comment (tkondrashov) [inline tf/train.py]: didn't touch this section cause it's brand new and it's not as clear how to split it up, but might revisit

File: tf/train.py [hunk 3]
```diff
@@ -32,33 +32,20 @@
...
-        chunks += get_chunks(d)
+    for dir in glob.glob(path):
+        chunks += glob.glob(f"{dir}*.gz")
     return chunks
 
 
 def get_latest_chunks(path, num_chunks, allow_less):
     chunks = get_all_chunks(path)
-    if len(chunks) < num_chunks:
-        if allow_less:
```
Comment (tkondrashov) [inline tf/train.py]: this whole section is repeated immediately underneath, not necessary

File: tf/train.py [hunk 4]
```diff
@@ -32,33 +32,20 @@
...
-            print("sorting {} chunks...".format(len(chunks)), end='')
-            chunks.sort(key=os.path.getmtime, reverse=True)
```
Comment (tkondrashov) [inline tf/train.py]: this section appears again on line 62, no need to run it twice. the only difference is the line `chunks = chunks[:num_chunks]` that appears on line 64 but not in this section, but it's fine for it to run later, it just takes a slice of the whole array

**Comment by tkondrashov** [comment]
made this PR a smaller section and added PRs for other sections of the file

**Comment by Tilps** [comment]
This is obsolete.

**Pull request closed without merging**